glue/clients/ds9norm.py

```diff
@@ -123,8 +123,8 @@ def __init__(self):
         self.stretch = 'linear'
         self.bias = 0.5
         self.contrast = 1.0
-        self.clip_lo = 5.
-        self.clip_hi = 95.
+        self.clip_lo = 0.
+        self.clip_hi = 100.
```

Comment:
It looks the same as what one would get in e.g. ds9 though, and I think it's going to be really hard to find a default that is esthetically pleasing for all datasets. I think that if we make the data limits and options to change the scale more obvious by putting them in the options widget, we could then include a step in the tutorial to show users how to change the stretch - what do you think?